plugins/harness-ci-cd/README.md

```diff
@@ -99,6 +100,26 @@ const serviceEntityPage = (
 
 4. Add required harness specific annotations to your software component's respective `catalog-info.yaml` file. Here is an example: https://github.com/harness/backstage-plugins/blob/main/examples/catalog-harness-cicd.yaml
 
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  # ...
+  annotations:
+    harness.io/pipelines: |
+      labelA: <harness_pipeline_url>
+      labelB: <harness_pipeline_url>
+  # here labelA / labelB denotes the value you will see in dropdown in execution list. Refer screentshot 1
+    harness.io/services: |
```

Comment:
I think we should convey that both of these are optional and only one annotation needs to be present at minimum